database/schema/13-banking.sql

```diff
@@ -0,0 +1,74 @@
+-- +migrate Up
+CREATE TABLE IF NOT EXISTS vipcoin_chain_banking_banking (
+    id                  SERIAL PRIMARY KEY NOT NULL,            -- banking id
+    asset               TEXT NOT NULL,                          -- banking asset
+    amount              BIGSERIAL,                              -- banking amount
+    kind                INT,                                    -- banking amount
+    extras              JSONB,                                  -- banking extras
+    timestamp           TIMESTAMP WITHOUT TIME ZONE NOT NULL,   -- banking timestamp
+    tx_hash             TEXT NOT NULL                           -- banking tx hash
+);
+
+CREATE TABLE IF NOT EXISTS vipcoin_chain_banking_system_transfer (
+    creator             TEXT NOT NULL,                      -- banking creator
+    wallet_from         TEXT NOT NULL,                      -- banking wallet from
+    wallet_to           TEXT NOT NULL,                      -- banking wallet to
+    asset               TEXT NOT NULL,                      -- banking asset
+    amount              BIGSERIAL,                          -- banking amount
+    extras              JSONB                               -- banking extras
+);
+
+CREATE TABLE IF NOT EXISTS vipcoin_chain_banking_set_reward_manager_address (
+    creator            TEXT NOT NULL,                       -- banking creator
+    address            TEXT NOT NULL                        -- banking address
+);
```

Comment:
```sql
CREATE TABLE IF NOT EXISTS vipcoin_chain_banking_reward_manager_address (
    address            TEXT NOT NULL                        -- banking address
);
```

add table please, current state of `reward_manager_address` and `set_reward_manager_address` table - will be a history